### PR TITLE
Split and Reuse the Release workflow

### PR DESCRIPTION
**Pull request by gastaldi**
- Needed for https://github.com/quarkiverse/quarkiverse-devops/issues/275. If you have customized the release process for your case, make sure they are properly adjusted, as this PR is automatically generated

### CODE REVIEW

File: .github/workflows/release-perform.yml
Comment (bpasson) [inline .github/workflows/release-perform.yml:1]: Think this is the content for release-prepare.yml

File: .github/workflows/release-prepare.yml
Comment (bpasson) [inline .github/workflows/release-prepare.yml:1]: Think this is the content for release-perform.yml

File: .github/workflows/release-perform.yml
Comment (gastaldi) [reply .github/workflows/release-perform.yml:1]: Indeed, I screwed up the names when creating the PRs and realized later on. Let me push a fix real quick

**Comment by bpasson** [comment]
Looks good @gastaldi don't have any special release procedure. I'll merge the PR.

**Pull request merged**